scripts/modules.js

```diff
@@ -145,3 +155,54 @@ export const modulesGet = async (id) => {
     );
   }
 };
+
+export const modulesArchive = async (id, unarchive = false) => {
+  const loadingSpinner = ora("Preparing request").start();
+
+  const moduleResponse = await apiClient.get({
+    path: `/v1/catalog/module/${id}`
+  });
+
+  loadingSpinner.stop();
+
+  if (!moduleResponse.ok) {
+    if (moduleResponse.status === 404) {
+      invalid(`Cannot find requested module with id ${id}.`);
+    } else {
+      invalid("Unable to get module. Please login and try again.");
+    }
+
+    return;
+  }
+
+  const verb = unarchive ? "unarchive" : "archive";
+
+  const module = await moduleResponse.json();
+
+  // Verify that the user understands which organization the modules are being published to.
+  const { ok } = await inquirer.prompt({
+    message: `Are you sure you want to ${verb} module: ${module.title}?`,
+    name: "ok",
+    type: "input",
+    default: "y/n"
+  });
+
+  if (ok.trim().toLowerCase() !== "y") {
```

Comment:
wondering by now if we should starting thinking about getting users to reenter a y/n command if they don't hit y or n. Maybe something like "That is not a valid option, please enter y/n". 

Not needed for this PR but just getting some ideas out there.